APTITUDE/APTITUDETEST.Data/Data/TestRepository.cs

```diff
@@ -317,10 +317,10 @@ public async Task<JsonResult> AddTestQuestions(TestQuestionsVM addTestQuestion)
                 _context.Add(testQuestionsToBeAdded);
                 _context.SaveChanges();
 
-                TestQuestionsCount testQuestionsCountToBeAdded = new TestQuestionsCount();
```

Comment:
Add this line again but don't assign it. like - TestQuestionsCount testQuestionsCountToBeAdded;